data/i18n/en/en.toml

```diff
@@ -9,6 +9,9 @@ other = "(auto-generated page)"
 [auto_generated_pageinfo]
 other = """<p>This page is automatically generated.</p><p>If you plan to report an issue with this page, mention that the page is auto-generated in your issue description. The fix may need to happen elsewhere in the Kubernetes project.</p>"""
 
+[bing_note]
+other = "The search results on this page are currently using Bing. Please note that Bing search may not be as effective as Google search at indexing websites where different query strings might be used, and the results may vary from your usual experience."
```

Comment:
I don't think it is a good idea to provide more information about the search tool besides the first sentence,
`The search results on this page are generated by Bing.`

Reply:
Sure, if it's not required than i will remove that :)